sympy/physics/control/lti.py

```diff
@@ -1,14 +1,24 @@
-from sympy import Basic, Mul, degree, Symbol, sympify
+from sympy import Basic, Mul, degree, Symbol, sympify, expand
+from sympy.core.numbers import Integer
 
 __all__ = ['TransferFunction',]
 
 
-class TransferFunction(Mul):
-    # this will subclass from `Basic`.
-    def __new__(cls, num, den):
-        obj = Mul.__new__(cls, num, 1/den)
-        obj.num = num
-        obj.den = den
+class TransferFunction(Basic):
+
+    def __new__(cls, num, den, var):
+
+        if not isinstance(var, Symbol):
+            raise TypeError("Last argument must be a complex Symbol.")
+        if den == 0:
+            raise ValueError("TransferFunction can't have a zero denominator.")
+
+        num, den = sympify(num), sympify(den)
```

Comment:
I don't think we should be simplifying these by default.

Reply:
I am using `sympify` to convert `int` to sympy integers. 
``` python
>>> tf3 = TransferFunction(3, 4, p);  type(tf3.num)
<class 'sympy.core.numbers.Integer'>
```

Reply:
It should be `_sympify` rather than `sympify` to prevent string inputs.